packages/react-storage/src/components/StorageBrowser/Views/LocationsView/LocationsView.tsx

```diff
@@ -61,6 +60,7 @@ export const LocationsMessage = (): React.JSX.Element | null => {
   ) : null;
 };
 
+// @ts-expect-error TODO: add Controls assignment
 const LocationsViewControls: LocationsViewControls = () => {
```

Comment:
Just remove the type assignment

Reply:
I tried removing it, but I think I'll have to update some of the other types too like for `LocationsView`. I think I can do this as a follow up though just so there isn't extra changes